datastructure/test/test_trie.py

```diff
@@ -76,3 +76,38 @@ def test_delete_word_does_not_exist(self):
         mytrie = create_trie(["hello", "hi"])
         result = mytrie.delete("hey")
         self.assertFalse(result)
+
+    def test_contains_prefix_word_exists(self):
+        mytrie = create_trie(["watch", "watching", "read", "reading"])
+        result = mytrie.contains_prefix("watch")
```

Comment:
try a prefix that isnt also a word, like watc or rea, since this should be true even if it isnt a word.